File: app/src/main/java/com/example/healthtracker/feature/references/view/ReferencesActivity.java
package com.example.healthtracker.feature.references.view;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.healthtracker.R;

public class ReferencesActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_references);

        ImageButton arrowBack = findViewById(R.id.arrow_refs);
        arrowBack.setOnClickListener(view -> {
            finish();
        });
    }
}

File: app/src/main/java/com/example/healthtracker/feature/references/view/ReferencesActivity.kt
package com.example.healthtracker.feature.references.view

import android.content.Intent
import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.text.Html
import android.text.method.LinkMovementMethod
import android.widget.ImageButton
import android.widget.TextView
import com.example.healthtracker.R

class ReferencesActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_references)

        val arrowBackButton: ImageButton = findViewById(R.id.arrow_refs)
        arrowBackButton.setOnClickListener {
            finish()
        }

    }

}